aws/sdk/integration-tests/s3/tests/query-strings-are-correctly-encoded.rs

```diff
@@ -15,13 +15,7 @@ use std::time::{Duration, UNIX_EPOCH};
 async fn test_s3_signer_query_string_with_all_valid_chars() {
     let (conn, rcvr) = capture_request(None);
     let sdk_config = SdkConfig::builder()
-        .credentials_provider(SharedCredentialsProvider::new(Credentials::new(
-            "ANOTREAL",
-            "notrealrnrELgWzOk3IfjzDKtFBhDby",
-            Some("notarealsessiontoken".to_string()),
-            None,
-            "test",
-        )))
+        .credentials_provider(SharedCredentialsProvider::new(Credentials::for_tests()))
```

Comment:
What do you think of adding a method to `SharedCredentialsProvider` like so:
```suggestion
        .credentials_provider(SharedCredentialsProvider::for_tests())
```

It's just about as clear, with a little less code.

Reply:
I would normally incorporate the suggestion but knowing `SharedCredentialsProvider` will cease to exist in #2122, we should probably leave it as is.